docker/clustering-script/get_clusters.py

```diff
@@ -6,9 +6,12 @@
 import clickhouse_connect.driver
 import clickhouse_connect.driver.client
 import numpy as np
-from sklearn.cluster import KMeans
+from sklearn.cluster import HDBSCAN
 from scipy.spatial.distance import cosine
-anthropic_client = anthropic.Anthropic()
+import dotenv
+
+dotenv.load_dotenv()
+anthropic_client = anthropic.Anthropic(api_key=os.getenv("LLM_API_KEY"))
```

Comment:
Either need to modify the Dockerfile to use this new env var or just use `ANTHROPIC_API_KEY` which it automatically pulls.